Makefile

```diff
@@ -158,20 +164,20 @@ acceptance: release/clean acceptance/pull docker-build release
 acceptance/run: acceptance/kind acceptance/load acceptance/setup acceptance/deploy acceptance/tests acceptance/teardown
```

Comment:
if the images aren't already on your machine this fails. `acceptance/pull` needs to be done after `acceptance/kind`